src/pages/Listeners.vue

```diff
@@ -70,21 +140,71 @@ export default {
       });
     },
     filterListeners(val) {
-      this.$store.dispatch('listeners/fetchAllUsers', {
+      console.log(val);
+      this.$store.dispatch('listeners/fetchAllListeners', {
         filterBy: {
           field: 'seminar_id',
           value: val,
         },
       });
     },
+    openConfirmDeleteModal() {
+      this.isConfirmDeleteModalOpen = true;
+    },
+    showCreateModal() {
+      this.isCreateModalOpen = true;
+    },
+    async showEditModal(id) {
+      await this.$store.dispatch('listeners/fetchListener', id);
```

Comment:
have to use [mapActions](https://vuex.vuejs.org/api/#mapactions)

Reply:
Done.